docs/spark/spark-configuration.md

```diff
@@ -185,4 +185,5 @@ df.write
 | snapshot-property._custom-key_    | null            | Adds an entry with custom-key and corresponding value in the snapshot summary  |
 | fanout-enabled       | false        | Overrides this table's write.spark.fanout.enabled  |
 | check-ordering       | true        | Checks if input schema and table schema are same  |
-
+| isolation-level | null | Desired isolation level for transaction.  For overwrites, `serializable` checks for concurrent inserts or deletes in destination partitions, `snapshot` checks for concurrent deletes. |
```

Comment:
I think it's unclear what this applies to in the context here. Does it only apply to merges? All Operations? What does "null" mean?